tests/cli/counter.sh

```diff
@@ -75,8 +51,8 @@ test02GetCounter() {
   assertFalse "no default count" $?
 }
 
-# checkAccount $COUNT $BALANCE
-# assumes just one coin, checks the balance of first coin in any case
+# CheckAccount $COUNT $BALANCE
```

Comment:
Uh... it was lower case due to the function name.  Also wrong function.  I'll fix it